gulpfile.js

```diff
@@ -0,0 +1,8 @@
+var gulp = require("gulp");
+var babel = require("gulp-babel");
+
+gulp.task("default", function () {
+  return gulp.src("*.js")
+    .pipe(babel())
+    .pipe(gulp.dest("dist"));
+});
```

Comment:
If you're only having a single compile step then you can just as well put it in your `package.json`. I.e. you can replace the `gulpfile` with a script `'build' : 'babel src/* --out-dir dist'` and have `babel` as a dev dependency. That does exactly the same as this, but doesn't require gulp.